src/utils/calculateCrc8.ts

```diff
@@ -0,0 +1,18 @@
+//CRC8を計算する. 初期値0xff, 多項式:x^8 + x^5 + x^4 + 1
+//参考:https://qiita.com/jamjam/items/7a3b80affe5ba6ecdead
```

Comment:
```suggestion
/**
 * 与えられたバイナリデータのCRC8ハッシュ値を計算する
 * - CRCレジスタ初期値: `0xff`
 * - 生成多項式: `x^8 + x^5 + x^4 + 1`(`0x31`)
 * - RefIn: なし(`false`)
 * - RefOut: なし(`false`)
 * - XorOut: なし(`0x00`)
 *
 * @see https://www.sunshine2k.de/articles/coding/crc/understanding_crc.html
 */
```
調べていてかなり混乱したので、このくらい厳密なコメントがあったほうがよさそうです。
また、Qiitaの方の記事は"こんな実装があるよ"ということはわかりますが、"なぜそれで正しいのか"が少なくとも私には分かりませんでした。こちらの記事を読んで初めて、CRCレジスタを用いた実装が一般的に説明されるものと等価であることが理解できました。